__tests__/game_test.js

```diff
@@ -1,117 +1,83 @@
 import { Game } from '../src/game';
+import { Board } from '../src/board';
 
 describe('Game', () => {
-
-  let game;
-  beforeEach(() => {
-    game = new Game;
-  });
+  //
+  // let game;
+  // beforeEach(() => {
+  //   game = new Game;
+  // });
 
   it('creates an instance of the Board class when initialized', () => {
+    let board = new Board([1, 2, 3, 4, 5, 6, 7, 8, 9])
```

Comment:
Super minor, but these can be `const` rather than let, as they don't change! It's [pretty easy](https://atom.io/packages/linter-eslint) to set up eslint with Atom!

Reply:
Thanks Dan. This has been amended

Reply:
And we've installed the package too. Thanks! :)